### PR TITLE
[CARBONDATA-2369] FAQ update related to carbon SDK scenario

### PR DESCRIPTION
**Pull request by ajantha-bhat**
[CARBONDATA-2369] FAQ update related to carbon SDK scenario

 - [ ] Any interfaces changed? no
 
 - [ ] Any backward compatibility impacted? no
 
 - [ ] Document update required? yes, updated

 - [ ] Testing done. NA

### CODE REVIEW

**Comment by sgururajshetty** [comment]
LGTM

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4197/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4508/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5366/

File: docs/faq.md [hunk 1]
```diff
@@ -182,3 +183,15 @@ select cntry,sum(gdp) from gdp21,pop1 where cntry=ctry group by cntry;
 ## Why all executors are showing success in Spark UI even after Dataload command failed at Driver side?
 Spark executor shows task as failed after the maximum number of retry attempts, but loading the data having bad records and BAD_RECORDS_ACTION (carbon.bad.records.action) is set as “FAIL” will attempt only once but will send the signal to driver as failed instead of throwing the exception to retry, as there is no point to retry if bad record found and BAD_RECORDS_ACTION is set to fail. Hence the Spark executor displays this one attempt as successful but the command has actually failed to execute. Task attempts or executor logs can be checked to observe the failure reason.
 
+## Why different time zone result for select query output when query SDK writer output? 
+SDK writer is an independent entity, hence SDK writer can generate carbondata files from a non-cluster machine that has different time zones. But at cluster when those files are read, it always takes cluster time-zone. Hence, the value of timestamp and date datatype fields are not original value.
+If you do not want to see according to time-zone, then set cluster's time-zone in SDK writer by calling below API.
```
Comment (gvramana) [inline docs/faq.md]: If wanted to control timezone of data while writing, then set cluster's time-zone in SDK writer by calling below API.

Comment (ajantha-bhat) [reply docs/faq.md]: done. will take this changes in #2198

**Comment by ajantha-bhat** [comment]
This will be handled in #2198.

No need of separate PR

**Pull request closed without merging**